app/helpers/report_helper.rb

```diff
@@ -146,4 +146,129 @@ def reports_group_title
+    data = summary_data('Basic Information')
+    data[:rows] = [
+      row_data('Title', "#{read_only}#{widget.title}"),
+      row_data('Descrption', "#{read_only}#{widget.description}"),
+      row_data('Active', widget.enabled),
+      row_data('Default', widget.read_only)
+    ]
+    miq_structured_list(data)
+  end
+
+  def widget_status(widget, sb_data, timezone)
+    return if sb_data[:wtype] == "m"
+
+    data = summary_data('Status')
+    glyphicon = widget.decorate.try(:secondary_icon)
+    current_status = glyphicon ? [{:icon => glyphicon, :value => widget.status}] : widget.status
+    data[:rows] = [
+      row_data('Current Status', current_status),
+      row_data('Last Run Time', h(widget.last_run_on ? format_timezone(widget.last_run_on, timezone, "view") : "")),
+      row_data('Message', h(widget.status_message))
+    ]
+    miq_structured_list(data)
+  end
+
+  def widget_sb_type_report(widget, sb_data)
+    case sb_data[:wtype]
+    when "r"
+      report_options(widget, sb_data)
+    when "c"
+      chart_options(widget)
+    when "m"
+      menu_shortcuts(widget)
+    end
+  end
+
+  def report_options(widget, sb_data)
+    data = summary_data('Report Options')
+    rows = []
+    if widget.resource
+      rows.push(row_data('Report Filter', h(widget.resource.name)))
+    else
+      rows.push(row_data('Report Filter', "*** %s ***" % _('Report no longer exists'), 'color_red'))
+    end
+    sb_data[:col_order].each_with_index do |col, idx|
+      rows.push(row_data("%s #{idx + 1}" % _('Column'), h(col)))
+    end
+    rows.push(row_data('Row Count', h(widget.row_count)))
+    data[:rows] = rows
+    miq_structured_list(data)
+  end
+
+  def chart_options(widget)
+    data = summary_data('Chart Options')
+    rows = []
+    if widget.resource
+      rows.push(row_data('Filter', h(widget.resource.title)))
+    else
+      rows.push(row_data('Filter', "*** %s ***" % _('Chart no longer exists'), 'color_red'))
+    end
+    data[:rows] = rows
+    miq_structured_list(data)
+  end
+
+  def menu_shortcuts(widget)
+    data = summary_data('Menu Shortcuts')
+    rows = []
+    widget.miq_widget_shortcuts.order("sequence").each_with_index do |_ws, wsi|
+      rows.push(row_data('', h(widget.description))) unless wsi == 0
+    end
+    data[:rows] = rows
+    miq_structured_list(data)
+  end
+
+  def widget_timer(schedule, timezone, sb_data)
+    if sb_data[:wtype] == "m"
+
+      data = summary_data('Timer')
+      rows = []
+      if schedule.kind_of?(MiqSchedule)
+        rows.push(row_data('Run At', h(schedule.run_at_to_human(timezone).to_s)))
+        run_time = schedule.next_run_on.blank? ? '' : h(format_timezone(schedule.next_run_on, timezone, "view"))
+        rows.push(row_data('Next Run Time', run_time))
+        data[:rows] = rows
+      else
+        data[:message] = "#{_('No timer is attached to this Widget, its contents will not be updated.')} #{_('Edit this Widget to configure a timer.')}"
```

Comment:
Is there a reason why this is two separately translated strings instead of just one?

Reply:
It was like that in the original I believe...I also think it's strange

Reply:
the translations were different for two strings printed in different lines. The `MiqStructuredList` component currently accepts just one message.